Source/ACE/Command/Handlers/DebugCommands.cs

```diff
@@ -779,7 +779,8 @@ public static void HandleListPlayers(Session session, params string[] parameters
             "Creates 1 of each weapon class in your inventory.")]
         public static void HandleWeapons(Session session, params string[] parameters)
         {
-            HashSet<uint> weaponsTest = new HashSet<uint>() { 93, 127, 130, 136, 136, 136, 148, 300, 307, 311, 326, 338, 348, 350, 7765, 12748, 12463, 31812 };
+            // HashSet<uint> weaponsTest = new HashSet<uint>() { 93, 127, 130, 136, 136, 136, 148, 300, 307, 311, 326, 338, 348, 350, 7765, 12748, 12463, 31812 };
+            HashSet<uint> weaponsTest = new HashSet<uint>() { 120, 134, 300, 307, 36561 };
```

Comment:
I did a quick set of testing enums to get rid of the magic numbers.   I was going to add them all but 32k enums makes the build time go to minutes instead of seconds.   We had discussed (Mogwai) of not wanting that large of a list of enums.   That said, if you know another way to accomplish that, I will be glad to do it.